### DISCUSSION TITLE
组件增加不随浏览器滚动条滚动的功能配置

### DISCUSSION THREAD

**Discussion by biuaxia** [ppoffice/hexo-theme-icarus#917] [Q&A]
**组件增加不随浏览器滚动条滚动的功能配置**

我在参考配置文档进行配置发现站长站点的右侧目录是不随浏览器滚动的。

烦请在配置项增加该功能，主要应用场景为长文阅读时没有目录，不方便返回到某一级标题去。

此功能应该是已经开发好了的吧，我观察站长站点是已经有了的（想通过yml方式实现，例如增加一个 `sticky:tue` 这种形式）。

<img width="960" alt="Snipaste_2021-08-16_08-52-51" src="https://user-images.githubusercontent.com/46131142/129498314-f03b2ef0-75a3-45f1-9a0b-8be8784fa7d1.png">

<img width="960" alt="Snipaste_2021-08-16_08-49-33" src="https://user-images.githubusercontent.com/46131142/129498320-9437d94c-c6c5-4928-83f0-0808aed6a846.png">

**Comment by ppoffice** [comment]
@biuaxia https://ppoffice.github.io/hexo-theme-icarus/Configuration/icarus%E7%94%A8%E6%88%B7%E6%8C%87%E5%8D%97-%E4%B8%BB%E9%A2%98%E9%85%8D%E7%BD%AE/#%E4%BE%A7%E8%BE%B9%E6%A0%8F

请设置`sticky`为`false`。
